pages/blog/[slug].tsx

```diff
@@ -45,6 +45,7 @@ export default function Post({ post, morePosts, preview, slug }: Post) {
   const postUrl = post?.slug
     ? `https://kedro.org/blog/${post.slug}`
     : 'https://kedro.org';
+  const mastodonContent = 'See more: https://social.lfx.dev/@kedro';
```

Comment:
Is there a way for the content to contain a link to the article itself? Like

"""
I read this article on the Kedro blog: {title} {url} by @kedro@social.lfx.dev

#kedro #kedroviz #python #pydata #datascience
"""

![image](https://github.com/kedro-org/kedro-website/assets/316517/1e782860-c36a-4d9a-8adf-b347cf94655f)

Reply:
Yeah of course! I managed to get it to this; I think hyperlinking the kedro@social.lfx.dev text though may be a separate task, there's no immediate way it seems to get it working 

<img width="597" alt="image" src="https://github.com/kedro-org/kedro-website/assets/35583819/76f622dd-165f-4f22-94d4-d63e58897278">

Reply:
The first few hashtags are static and hardcoded (kedro, kedroviz, python, etc) and included in all posts that'd be shared, whilst the last few are auto-generated using the post's categories (e.g. a Post Category 'Datascience, Databricks' would become #datascience #databricks')

Reply:
![IMG_20230915_231057](https://github.com/kedro-org/kedro-website/assets/316517/e080d223-0555-4081-bad0-10246f43939f)

Tried from mobile, works perfectly!